framework/set/provisioning/providers/harvester/harvesterProviders.go

```diff
@@ -0,0 +1,88 @@
+package harvester
+
+import (
+	"github.com/hashicorp/hcl/v2"
+	"github.com/hashicorp/hcl/v2/hclsyntax"
+	"github.com/hashicorp/hcl/v2/hclwrite"
+	"github.com/rancher/tfp-automation/config"
+	"github.com/rancher/tfp-automation/defaults/resourceblocks/nodeproviders/harvester"
+	"github.com/rancher/tfp-automation/framework/set/defaults"
+	"github.com/zclconf/go-cty/cty"
+)
+
+const (
+	cloudCredential = "rancher2_cloud_credential"
```

Comment:
This is already defined in `framework/set/defaults/defaults.go`, please reference that instead.